dev-scripts/serve-docker

```diff
@@ -23,6 +23,7 @@ set +x
 set -x
 
 PS_SHARED_SECRET=somepassword
+LITESTREAM_RETENTION=${LITESTREAM_RETENTION:-72h}
```

Comment:
We can skip the retention setting in this file, as this script is just for development.